datafusion-examples/examples/to_char.rs

```diff
@@ -125,14 +125,14 @@ async fn main() -> Result<()> {
 
     assert_batches_eq!(
         &[
-            "+------------+",
-            "| t.values   |",
-            "+------------+",
-            "| 2020-09-01 |",
-            "| 2020-09-02 |",
-            "| 2020-09-03 |",
-            "| 2020-09-04 |",
-            "+------------+",
+            "+-----------------------------------+",
+            "| arrow_cast(t.values,Utf8(\"Utf8\")) |",
```

Comment:
These differences are due to the fact that `arrow_cast` is just a normal function now rather than a special case in the parser. Thus the naming reflects normal function naming

Reply:
Interesting I got stuck implementing the simpliy function because I thought it should convert `arrow_cast(t.values,Utf8(\"Utf8\"))` to `t.values` and other similar cases as well.

Reply:
Yeah -- this is pretty tricky. `arrow_cast` was quite special in the parser, so now that it is handled like a normal function it has the same (somewhat strange) function effect of column naming